client/source/js/modules/admin/manage-users-ctrl.js

```diff
@@ -0,0 +1,21 @@
+define(['./module', 'angular', 'underscore'], function (module, angular, _) {
+  'use strict';
+  module.controller('AdminManageUsersController', function ($scope, $http, info, users, UserManager, modalService) {
+    $scope.users = _(users.data.users).filter(function (u) {
+      return u.email != UserManager.data.email;
+    });
+
+    $scope.deleteUser = function (user) {
+      modalService.confirm(function(){
+        user_id = user.id;
```

Comment:
Wahhh, fix it fast & never tell anyone. You introduced a global variable :)

hmm what is the benefit of using user_id instead of user.id?
hmm user_id is not really camelCase :smiley: